docs/_posts/1871-05-19-Louis-Rau.md

```diff
@@ -5,27 +5,27 @@ author: Louis Rau
 layout: post
 place: Brussel
 lang: 'de'
-signature: 'Eure Ludwig'
+signature: 'Euren Ludwig'
 scan_image: '1871-05-19_Brussels_Ludwig_Rau.jpg'
 note: (Au dos de [la lettre des 16 et 17 Mai 1871](/1871/05/17/Louis-Guillier.html))
 ---
 
 Meine Theuren,
 
 Beifolgend 2 Bf von
-<span data-lang="fr">Guillier</span>; es ist schauderschrift mir es
+<span data-lang="fr">Guillier</span>; es ist schauderhaft wie es
 in <span data-lang="fr">Paris</span> ausschaut & man weis
-gar nicht wan das reden wird.
+gar nicht wan̄ das enden wird.
 
-Einstweilen kann ich mich sie
-wie wenig beschaufliegen, und
-denke ich d?dung wenige an unse
-<span data-lang="fr">Schlamassel</span> : — Wir sehen Euen
+Einstweilen kann ich mich hier
+nur wenig beschäftigen, und
+denke ich dadurch weniger an unser
+<span data-lang="fr">Schlamassel</span> : — Wir sehnen Euren
 guten Negrenschen betreffs Landerhaus
-& Wohldbefinden entzogen. — <span data-lang="fr">Hugos</span>
-gehen heute Abend ans <span data-lang="fr">Chatou</span> weil
-sie es hier nicht mehr ansfalten
+& Wohlbefinden entgegen. — <span data-lang="fr">Hugos</span>
+gehen heute Abend nach <span data-lang="fr">Chatou</span> weil
+sie es hier nicht mehr aushalten
 können.
 
-<span data-lang="fr">Kunst</span> wohl & grüßt
```

Comment:
I still think it might be "Kunst". David Kunst (https://www.geni.com/people/David-Klimat-or-Kuntz-Kunst/6000000023926284498), Ludwig's brother-in-law, lived in Brussels at the time, and this is where Ludwig is writing from as well.

Reply:
Tempting but not probable. A name at this position doesn't make sense semantically and the construction is not grammatical at all. My reading is not a common phrase in modern German but it can be understood as something like 'know that I'm well".

Reply:
I was just surprised at the use of "grüßt mit mir". Would that be a standard expression?

Reply:
Not in today's language, no. But greeting formulas in letters have changed a lot in German since then (much more than in French, it seems to me).

Reading “Kunst” would require rereading the following word which is, however, quite clearly written, so much more likely “wohl” than anything else.

Reply:
I'm still not entirely satisfied with the closing formula. That first letter could be S or P as well. I'll let you know when I have a better version :)

Reply:
Thanks :-)